terraform/sandbox/main.tf

```diff
@@ -63,6 +63,7 @@ module "egress-space" {
   cf_restricted_space_name = local.cf_space_name
   deployers = [
     var.cf_user,
-    "ryan.ahearn@gsa.gov"
+    "ryan.ahearn@gsa.gov",
+    "steven.reilly@gsa.gov"
```

Comment:
I spent a moment thinking about ways to pull this data in from a GitHub team or CF OrgManagers or something, but I think that'd be over-engineering it.

Reply:
I _really_ wanted to set this list to be equal to the SpaceDeployers on the target space, but terraform doesn't support querying that information.